src/drawer.multicanvas.js

```diff
@@ -332,11 +332,15 @@ WaveSurfer.util.extend(WaveSurfer.Drawer.MultiCanvas, {
 
     getImage: function(type, quality) {
         // combine all available canvasses together
-        var availableCanvas = '';
+        var availableCanvas = [];
         for (var i in this.canvases) {
             var getEntry = this.canvases[i].wave.getContext('2d');
-            availableCanvas += getEntry.canvas.toDataURL(type, quality);
+            availableCanvas.push(getEntry.canvas.toDataURL(type, quality));
```

Comment:
1. The `entry` object already includes a cached version of the 2d context.
2. The `canvas` property of the 2d context is just a reference to the underlying canvas HTMLElement. So at the moment this code is unnecessarily complex. Please do instead: `availableCanvas.push(entry.wave.toDataURL(type, quality));` – No need for `getEntry`